### DISCUSSION TITLE
Testing  Slot persistence

### DISCUSSION THREAD

**Discussion by Tetlys** [Vdauphin/HeartsAndMinds#1363] [Question and answer]
Hey, we were just testing out the persistent save for players. Seems great, anyway to disable it currently as I noticed no Param for this as of yet.

**Comment by Vdauphin** [comment]
Hello,

No way to disable it and I don't think I will add any param for that
The main idea is to translate the vehicle saving system (position, damage...) to the player
But I plan to have a setting that each player will have a seperate saving per slot. Now, players are sharing the slot saving 
See this for more info https://github.com/Vdauphin/HeartsAndMinds/discussions/1324#discussioncomment-2412746

Cheers

**Reply by Tetlys** [reply]
So slot persistence in the future will be player specific, not shared as it is now?

**Reply by Vdauphin** [reply]
Both will be available thought mission parameters

**Reply by Tetlys** [reply]
Cool, so right now the only issue we have is when someone DC's in a random town or whatever, then the next person who joins, the next day say, is left alone out in the woods. Anyway we could add a Return to base option for the first 30 seconds of spawning or somthing?

**Reply by Vdauphin** [reply]
Well he could disconnect and select an other slot
Or you still have the respawn button

**Reply by Tetlys** [reply]
Changing the slot doesn't really help, still leaves the other slot out in the field.  And enabling respawn button isn't an option. We have it disabled to stop people respawning the second they get hit and wasting tickets.

**Reply by Vdauphin** [reply]
You could add a scroll menu to let people go back to base by using `btc_respawn_marker`

**Comment by Tetlys** [comment]
For anyone who wants it, This will allow a tp back to base to anyone outside of spawn for the first 30 seconds. :) Just throw the code in our OnPlayerRespawn.sqf

```
if (player distance (getMarkerPos "btc_respawn_marker") >500) then{

_ActionID1 = player addAction  
    [  
    "Respawn", // title  
    {  
    params ["_target", "_caller", "_actionId", "_arguments"]; // script  
    player removeAction _actionID;
	player setPos (getMarkerPos "Respawn_west");
	},  
    nil,  // arguments  
    1.5,  // priority  
    true,  // showWindow  
    true,  // hideOnUse  
    "",   // shortcut  
    "",   // Conditions
    -1,   // radius  
    false //Allow when Uncon 
    ];

sleep 30;
player removeAction _actionID1
}

```
